.github/workflows/ci.yml

```diff
@@ -12,10 +12,13 @@ on:
     branches: ['*']
   push:
     branches: ['*']
-    tags: [v*]
+    tags: [v*, v*]
```

Comment:
Why twice?

Reply:
It's a quirk in sbt-github-actions that I haven't managed to ask about.